src/viewsProto2/Speak/Speak.js

```diff
@@ -51,7 +51,7 @@ function Speak() {
   const [load_cnt, set_load_cnt] = useState(0);
 
   const [resultnext, set_resultnext] = useState("start");
-  const [resultnextlang, set_resultnextlang] = useState("ta");
+  const [resultnextlang, set_resultnextlang] = useState("hi");
```

Comment:
please check it has impact on our current task

Reply:
Updated it